src/client/components/ReviewAndSubmitSection.jsx

```diff
@@ -35,28 +37,81 @@ class ReviewAndSubmitSection extends React.Component {
     } else {
       content = (<div>
         <p>Please ensure all of your information is correct before submitting your application.</p>
-        <NameAndGeneralInfoSection reviewSection/>
-        <VAInformationSection reviewSection/>
-        <AdditionalInformationSection reviewSection/>
-        <DemographicInformationSection reviewSection/>
-        <VeteranAddressSection reviewSection/>
-        <InsuranceInformationSection reviewSection/>
-        <MedicareMedicaidSection reviewSection/>
-        <ServiceInformationSection reviewSection/>
-        <AdditionalMilitaryInformationSection reviewSection/>
-        <FinancialDisclosureSection reviewSection/>
-        <SpouseInformationSection reviewSection/>
-        <ChildInformationSection reviewSection/>
-        <AnnualIncomeSection reviewSection/>
-        <DeductibleExpensesSection reviewSection/>
-        <div className="input-section">
-          <a href="#">Upload documents</a>
-        </div>
+
+        <ReviewCollapsiblePanel
+            sectionLabel="NameAndGeneralInfoSection"
```

Comment:
Perhaps add a TODO here to come back and update the names of these sections? I doubt we'll forget but you never know :)